libs/application/templates/general-fishing-license/src/forms/GeneralFishingLicenseForm/applicantInformationSection.ts

```diff
@@ -22,8 +22,15 @@ export const applicantInformationSection = buildSection({
           backgroundColor: 'white',
           disabled: true,
           required: true,
-          defaultValue: (application: GeneralFishingLicense) =>
-            application.externalData?.nationalRegistry?.data?.fullName,
+          defaultValue: (application: GeneralFishingLicense) => {
+            const company =
+              application.externalData.companyNationalRegistry.data.name
```

Comment:
I suggest simplify the property naming by using `identity` instead.
```suggestion
              application.externalData.identity.data.name
```